lib/socket.js

```diff
@@ -65,7 +65,7 @@ class Transport extends EventEmitter {
     this._buffer += chunk;
 
     try {
-      this._buffer = jsrs.parseNetworkPackets(this._buffer, packets);
+      this._buffer = json5.parseNetworkPackets(this._buffer, packets);
```

Comment:
I don't like the idea of this PR overall because it is not totally correct to call our old JSRS format JSON5, its idea was to support the latest specification of JavaScript to be used as a serialization format, and it supports [6th version of standard](http://www.ecma-international.org/ecma-262/6.0/) at the moment, possibly even 7th version (I haven't checked this yet), while JSON5 is specified to use version 5.1 of standard in its core (that's the reason this format is called JSON5).

I am sorry that I didn't spot this fact earlier, that JSON5 is limited with language features from version 5.1, meaning it is not entirely correct to call our parser/serializer JSON5-compliant.